lib/frontend/terminal_view.dart

```diff
@@ -311,7 +310,8 @@ class _TerminalViewState extends State<TerminalView> {
     _lastTerminalWidth = termWidth;
     _lastTerminalHeight = termHeight;
 
-    widget.terminal.resize(termWidth, termHeight);
+    widget.terminal
+        .resize(termWidth, termHeight, _width!.floor(), _height!.floor());
```

Comment:
There's an application that uses `ws_xpixel` and `ws_col` to determine the size of a single cell.

https://github.com/Canop/broot/blob/685836c7a465f992e6a4f2a3f4c2feca23df3403/src/display/cell_size.rs#L35

Maybe we use `termWidth * _cellSize.cellWidth` rather than `_width!.floor()` ?

Reply:
I agree with this, let me change it soon.

Reply:
I changed those lines, please have a look.